### DISCUSSION TITLE
Can´t farm. the app does not start properly

### DISCUSSION THREAD

**Discussion by lucasCava** [Chia-Network/chia-blockchain#7918] [Support]
I have Chia 1.2.3 version. The full node and wallet works fine. Plots does not start properly. It shows a circle spinning. Farm does not show: "Your Farm Overview". Pool says "Loading Plot NFTs" and it remains that way. i Can´t farm, or start doing the new plots.

**Comment by lucasCava** [comment]
i deleted the database... and started from scrach...  the problem has been solved. 
Thanks for not answering.

**Reply by keliew** [reply]
Thanks for reading.

https://github.com/Chia-Network/chia-blockchain/wiki/Windows-Tips-and-Tricks#2-take-a-good-look-at-log-files
https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide#how-to-tell-if-chia-is-working-on-windows-gui
https://github.com/Chia-Network/chia-blockchain/wiki/Windows-Tips-and-Tricks#5-forever-connecting-to-wallet

Usually the log would give some idea of what went wrong, and if I'm not wrong, it's your corrupted `config.yaml`, since your node and wallet are sync-ed, it shouldn't be a database problem.
